core/src/consensus/light_client/mod.rs

```diff
@@ -44,10 +44,11 @@ impl ClientState {
 /// All data for updating a light client up to Nth block.
```

Comment:
For the " Fix invalid light client core algorithm " commit, please write these things in your commit message:
* What the problem was: number was not verified.
* How you fixed the problem: include a raw header in the "UpdateHeader".